fluentui_listitem/src/main/java/com/microsoft/fluentui/tokenized/listitem/ListItem.kt

```diff
@@ -620,10 +621,13 @@ object ListItem {
                 interactionSource = interactionSource
             )
             val horizontalPadding = getListItemTokens().padding(Medium)
-            val leadPadding = if(leadingAccessoryView == null){
+            val leadPadding = if (leadingAccessoryView == null) {
                 PaddingValues(start = horizontalPadding, end = horizontalPadding)
-            }else {
-                PaddingValues(start = getListItemTokens().padding(size = GlobalTokens.SpacingTokens.None), end = horizontalPadding)
+            } else {
+                PaddingValues(
+                    start = getListItemTokens().padding(size = GlobalTokens.SpacingTokens.None),
```

Comment:
Should we keep ListItemInfo in the token functions as discussed for all controls?

Reply:
Yes, Let's take it as separate PR for all the remaining controls.